recipes/LMRv2p1_MCruns_ensemble_ungridded/recipe.py

```diff
@@ -0,0 +1,50 @@
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern, MergeDim
+from pangeo_forge_recipes.recipes.xarray_zarr import XarrayZarrRecipe
+
+variables = ['gmt_full', 'posterior_climate_indices_full', 'shmt_full', 'nhmt_full']
+
+
+def make_url(time, variable):
+    pair = variable.rsplit('_', 1)
+    stem = 'https://www.ncei.noaa.gov/pub/data/paleo/reconstructions/tardif2019lmr/v2_1/'
+    nc_file = f'{pair[0]}_MCruns_ensemble_{pair[1]}_LMRv2.1.nc'
+    url = stem + nc_file
+    return url
+
+
+# the full time series is in each file, each of which is between ~300 mb and ~3 Gb
+time_concat_dim = ConcatDim('time', [0])
+pattern = FilePattern(make_url, time_concat_dim, MergeDim(name='variable', keys=variables))
+
+
+# ensures that lat and lon coords get labeled as simply 'lat' and 'lon'
+def postproc(ds):
+    _variables = [_var for _var in ds.data_vars.keys() if 'bound' not in _var]
```

Comment:
👋 @jordanplanders, `pre-commit` noticed that `_variables` is never used. Can it be deleted, or is the linter missing something?

```suggestion
```